### PR TITLE
[Link update] Update Link Usage, Style, and Accessibility tabs

### PR DESCRIPTION
**Pull request by thyhmdo**
Closes # https://github.com/carbon-design-system/carbon-website/issues/3537
Closes # https://github.com/carbon-design-system/carbon-website/issues/3550

Update Link Usage, Style, and Accessibility tabs

### CODE REVIEW

File: src/pages/components/link/usage.mdx [hunk 1]
```diff
@@ -81,86 +88,167 @@ be used for navigational actions.
...
 <Column colLg={8}>
 
-![link anatomy image](images/01_link_anatomy_608.png)
+![Link anatomy image](images/link-usage-anatomy.png)
 
 </Column>
 </Row>
 
-1. **Label:** Communicates what is being linked to.
+1. **Link text:** Communicates what is being linked to
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:96]:
```suggestion
1. **Link text:** Communicates what is being linked to.
```

File: src/pages/components/link/usage.mdx [hunk 2]
```diff
@@ -81,86 +88,167 @@ be used for navigational actions.
...
+2. **Icon (optional)** Indicates the destination or type of content being
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:97]:
```suggestion
2. **Icon (optional):** Indicates the destination or type of content being.
```

File: src/pages/components/link/usage.mdx [hunk 3]
```diff
@@ -81,86 +88,167 @@ be used for navigational actions.
...
-location, it is important that their labels accurately reflect the content users
-will find at the link destination. Use meaningful labels for links and avoid
-terms like "click here" or the web address itself.
+#### Link text
+
+- Use distinctive and meaningful link text.
+- Ensure that links accurately reflect the content users will find at the link
+  destination.
+- If your design uses generic link names such as “read more,” consider making
+  them unique. See
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:156]:
```suggestion
  them unique. See the
```

Comment (laurenmrice) [reply src/pages/components/link/usage.mdx:156]: I would link to the anchor link section "Ensure link context" that this text is referencing.

File: src/pages/components/link/usage.mdx [hunk 4]
```diff
@@ -81,86 +88,167 @@ be used for navigational actions.
...
+  [accessibility](https://carbondesignsystem.com/components/link/accessibility)
+  tab for more information.
+- Avoid using the same link text for different destinations on the same page and
+  try to differentiate between links by using unique text for each.
+- Ensure links are understandable but not excessively long, especially if they
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:161]: > Ensure links are understandable but not excessively long, especially if they wrap unless they are used inline.

Does this mean if long links wrap to multiple lines, they should be inline links and not standalone links?

File: src/pages/components/link/usage.mdx [hunk 5]
```diff
@@ -81,86 +88,167 @@ be used for navigational actions.
...
+For further content guidance, see Carbon's
+[content guidelines](https://carbondesignsystem.com/guidelines/content/overview/).
 
-## Behaviors
+## Universal behaviors
+
+## States
+
+List has six main states – enabled (unvisited), hover, focus, active, visited,
+and disabled. Learn more about states on the style tab.
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:181]:
```suggestion
and disabled. Learn more about states on the [style](https://carbondesignsystem.com/components/link/style) tab.
```

File: src/pages/components/link/usage.mdx [hunk 6]
```diff
@@ -81,86 +88,167 @@ be used for navigational actions.
...
 
-Users can open a link by clicking anywhere along the link text or on the
-associated icon.
+Users can open a link by clicking anywhere on the link text or on the associated
+icon.
 
 #### Keyboard
 
 Users can open a link by pressing `Enter` while the link has focus. For
 additional keyboard interactions, see the Accessibility tab.
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:202]:
```suggestion
additional keyboard interactions, see the [Accessibility](https://carbondesignsystem.com/components/link/accessibility#keyboard-interactions) tab.
```

File: src/pages/components/link/usage.mdx [hunk 7]
```diff
@@ -81,86 +88,167 @@ be used for navigational actions.
...
+Both standalone and inline links are clickable across the entire width of the
+text. Standalone link with an icon has an expanded clickable area that includes
+the icon.
 
-## Link variants
+<Row>
+<Column colLg={8}>
 
-### Standalone link
+![Standalone and inline link with their clickable areas](images/link-usage-clickable.png)
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:223]:
```suggestion
![Standalone and inline links with their clickable areas](images/link-usage-clickable.png)
```

File: src/pages/components/link/usage.mdx [hunk 8]
```diff
@@ -174,33 +262,112 @@ icons.
...
+</Column>
+</Row>
+
+<Caption>Example shows unvisited and visited links</Caption>
 
 ### Links that trigger actions
 
 Some links trigger actions to aid task completion in addition to navigation.
 These links should still serve a navigation purpose. A common example is linking
 phone numbers so clicking the website automatically opens and calls the phone
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:297]:
```suggestion
phone numbers, so clicking the website automatically opens and calls the phone
```

File: src/pages/components/link/usage.mdx [hunk 9]
```diff
@@ -174,33 +262,112 @@ icons.
...
-number when clicked. The label and any accompanying icons should make it clear
+number when clicked. The link text and any accompanying icons should clarify
 what action will be triggered and where the user will be directed.
 
+<Row>
+<Column colLg={8}>
+
+![Example shows inline links for email](images/link-usage-modifiers-2.png)
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:304]: The word GitHub in this image needs a capital H. And there should not be a space between the word and the period.
Screenshot 2024-02-09 at 2 23 11 PM

File: src/pages/components/link/usage.mdx [hunk 10]
```diff
@@ -174,33 +262,112 @@ icons.
...
+  </DoDont>
+</DoDontRow>
+
+### Link grouping
+
+<InlineNotification>
+
+**Code unaivailibility** The following specs are not developed for the link
+component but are recommended by design as the proper distance between grouped
+links. View [style](https://carbondesignsystem.com/components/link/style/) tab
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:331]:
```suggestion
links. View the [style](https://carbondesignsystem.com/components/link/style/) tab
```

File: src/pages/components/link/usage.mdx [hunk 11]
```diff
@@ -174,33 +262,112 @@ icons.
...
+![Do not use buttons when actions serve as a navigation purpose](images/link-usage-modifiers-dont.png)
+
+  </DoDont>
+</DoDontRow>
+
+### Link grouping
+
+<InlineNotification>
+
+**Code unaivailibility** The following specs are not developed for the link
```
Comment (laurenmrice) [inline src/pages/components/link/usage.mdx:329]:
```suggestion
**Code unavailability** The following specs are not developed for the link
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks really good Thy! Just some minor typos:

File: src/pages/components/link/usage.mdx (hunk 4 above)
Comment (thyhmdo) [reply src/pages/components/link/usage.mdx:161]: I didn't check on this statement. I was what you said before we decided to have standalone links wrapped as well. We can cut out the used inline part

**Review by alisonjoseph — APPROVED** [review]

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

**Review by laurenmrice — APPROVED** [review]
:rocket:

**Pull request merged**